### ISSUE TITLE
Update CNCF Social Media Guidelines to be broader

### ISSUE THREAD

**Issue by caniszczyk** [cncf/foundation#895]
The CNCF Social Media guidelines were originally created and focused on Twitter but we now use multiple social channels along with CNCF projects... from Twitter/X, LinkedIn, Instagram, Facebook, Mastadon, Bluesky and more!

We should update them to ensure they apply to all social media platforms.

**Comment by Nyalwendo** [comment]
Yes we should update them, but  what is the meaning of X, LinkedIn?

**Comment by k8tgreenley** [comment]
@Nyalwendo I will be providing updates to our social media guidelines.

**Comment by Nyalwendo** [comment]
Okay, Thanks  k8tgreenley.

**Comment by shwetd19** [comment]
Hey @k8tgreenley @Nyalwendo is this still open to work ?

**Comment by k8tgreenley** [comment]
@shwetd19 our internal team is working on the update
